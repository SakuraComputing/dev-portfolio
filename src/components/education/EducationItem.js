import React from 'react';

const EducationItem = (props) => {
    return (
        <div>
            <div className="item">
                <h3 className="title"><i className="fa fa-graduation-cap"></i>{props.educationitem.qualification}
                </h3>
                <h4 className="university">{props.educationitem.institution}<span className="year">{props.educationitem.year}</span></h4>
            </div>
        </div>
    );
};

export default EducationItem;
