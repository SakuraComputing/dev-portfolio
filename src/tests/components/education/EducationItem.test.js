import React from 'react';
import { shallow } from 'enzyme';
import EducationItem from '../../../components/education/EducationItem';

describe('EducationItem Section', () => {
    it('should render correctly', () => {
        const educationitem = jest.fn();
        const wrapper = shallow(<EducationItem educationitem={educationitem}/>);
        expect(wrapper).toMatchSnapshot();
    });
});