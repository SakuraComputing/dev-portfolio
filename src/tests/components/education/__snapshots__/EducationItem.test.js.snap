// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EducationItem Section should render correctly 1`] = `
<div>
  <div
    className="item"
  >
    <h3
      className="title"
    >
      <i
        className="fa fa-graduation-cap"
      />
    </h3>
    <h4
      className="university"
    >
      <span
        className="year"
      />
    </h4>
  </div>
</div>
`;
